zurichat/zc_plugin_duty_shift_dotnet
# Internal data integration with zccore db

#61 
**Test Step**

- Visit https://tracker.zuri.chat/
- On the top navbar, click on ZcCoreTest. (This uses the read endpoint to get currently saved data)

In the case where the above step doesn't give the expected result, here is the direct [link ](https://zccore.herokuapp.com/data/read/6131f73d569dbbb7ce5b4fc5/IntegrationTest/612a3a914acf115e685df8e3)=> https://zccore.herokuapp.com/data/read/6131f73d569dbbb7ce5b4fc5/IntegrationTest/612a3a914acf115e685df8e3

Note: api.zuri.chat/plugins/register returned 404 during testing. The test endpoint (zccore.herokuapp.com) was used as it returns result for /plugins/register endpoint.

**Included in this PR**

- ReadData Implementation : created an implementation to get data from the core db without the filter option. Location => App/Data/ZcCoreDataService/ReadData
- PostData Implementation : created an implementation to post single data entry (bulk_write: false) to the core db. Location => App/Data/ZcCoreDataService/PostData
- Created ZccoreDataIntTest controller, ZcCoreTestModels folder and FetchDataCopy.js to test the above implementation. (These files will be deleted after review)
